dascore/io/netcdf/utils.py

```
@@ -0,0 +1,653 @@
+        # Try to get dimension order from DIMENSION_LIST
+        if "DIMENSION_LIST" in data_var.attrs:
+            dim_list = data_var.attrs["DIMENSION_LIST"]
+            for ref_array in dim_list:
+                try:
+                    ref = ref_array[0]
+                    dim_scale = h5file[ref]
+                    dim_name = dim_scale.name.strip("/")
+                    coord_order.append(dim_name)
+                except Exception:
+                    # If we can't resolve reference, we'll fall back to discovery
+                    pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)